spec/features/hiring_staff_can_publish_a_vacancy_spec.rb

```diff
@@ -653,6 +653,47 @@
         end
       end
 
+      context 'editing the supporting_documents' do
+        let(:feature_enabled?) { true }
+        let(:document_upload) { double('document_upload') }
+
+        before do
+          allow(DocumentUpload).to receive(:new).and_return(document_upload)
+          allow(document_upload).to receive(:upload)
+          allow(document_upload).to receive_message_chain(:uploaded, :web_content_link).and_return('test_url')
+          allow(document_upload).to receive_message_chain(:uploaded, :id).and_return('test_id')
+          allow(document_upload).to receive(:safe_download).and_return(true)
+        end
```

Comment:
Do we need this block if the document upload is not re-tested?

Reply:
Good catch!